src/services/students/victoria-kondratska.ts

```diff
@@ -0,0 +1,108 @@
+import IAnimationConfig from "../../models/animation-config";
+import { IConfigLoaderService } from "../config-loader-interface";
+import { Colour } from '../../models/colour';
+import IFrame from '../../models/frame';
+import IPixelState from '../../models/pixel-state';
+
+export class VictoriaKondratskaConfigLoaderService implements IConfigLoaderService {
+
+    public getStudentName(): string {
+        return 'Victoria "Vitya" Kondratska';
```

Comment:
This will probably overflow, could you check please ?

Reply:
fixed that!